### DISCUSSION TITLE
Should the author of a post be able to moderate replies in the thread?

### DISCUSSION THREAD

**Discussion by robrwo** [mastodon/mastodon#22443] [General]
I'm thinking that it would be useful for the user who created a post to be able to flag any replies in a thread.

The flag could mark them as "Off topic" or "Abusive" or even "Content Warning".  Anyone viewing the thread would see a note that there are additional replies flagged by the original author. One could then click on a link/button to see those replies.

When one views those posts, there could be something that indicates that they were flagged.

A user might also be able to have a setting that lets them ignore flagged messages and not hide them when viewing a thread.

This would allow someone who creates a thread to be a moderator for replies that they consider disruptive. 

There wouldn't be any appeals, though the original poster could un-flag a message if they wanted to.

It allows decentralised moderation without censorship. It's also a way to keep people from disrupting a conversation.

**Comment by thefuturebird** [comment]
I'm not thrilled by such an option but not against it either. My concern with every user moderating individual responses is that this becomes a new vector for types of abuse. Eg. post something inflammatory then "moderate" anyone who takes the time to offer a kind and reasonable reply. 

I do think it is fair for anyone to turn off replies to their post. 

It would be nice if I could block all replies to my post not on some _domain white list_. I like being on an open server, small new instance have a lot to offer, but most of the racist spam I get is also from these domains. "white list only replies" would be a nice option for ... more sensitive posts.
